app/components/EditEntry.js

```diff
@@ -57,22 +91,66 @@ export default class EditEntry extends React.Component {
   };
 
   render() {
-    const { id, entry } = this.state;
+    const { id, entry, showColorPicker: showColorPicker } = this.state;
+
+    const defaultBgColor = '#fff';
```

Comment:
Should this be inserted into the array of options?

Reply:
This is just used to show that the entry doesn't have a color. Users can type this color manually if they'd like to use it, though.